### DISCUSSION TITLE
15 backups / Day while 1 line Backup Crontab -e

### DISCUSSION THREAD

**Discussion by DominArsen** [GameServerManagers/LinuxGSM#3361] [Q&A] [answered]
Hello,

today I put the following command line in "crontab-e"

```
#Automatic server start
@reboot / home / vhserver / vhserver monitor> /home/vhserver/log/CronDemarrageServeur.log 2> & 1
#every 5mins Test monitor + restart server if Valheim crashes
* / 5 * * * * / home / vhserver / vhserver monitor> /home/vhserver/log/CronMonitor.log 2> & 1
#every 30mins update
* / 30 * * * * / home / vhserver / vhserver update> /home/vhserver/log/CronUpdate.log 2> & 1
#Backup Valheim every evening at 4h00
* 4 * * * / home / vhserver / vhserver backup> /home/vhserver/log/CronBackup.log 2> & 1
#MAJ Valheim every evening at 4.30am
30 4 * * * / home / vhserver / vhserver force-update> /home/vhserver/log/cronUpdateForce.log 2> & 1
#Update LGSM server every Monday at 4:45 am
45 4 * * 1 / home / vhserver / vhserver update-lgsm> /home/vhserver/log/CronUpdateLGSM.log 2> & 1
#Restart server every evening at 5:00 a.m.
* 5 * * * / home / vhserver / vhserver restart> /home/vhserver/log/CronRestart.log 2> & 1
```

My problem is that when I look at the backup I created 15 in 1 hour while I only have one command ..
[https://i.ibb.co/D8MSxvR/Capture-d-cran-2021-03-13-125901.png](url)

Can you help me so that there is only 1 Backup / Day before the update?

**Comment by EugeneGGHQ** [comment, marked as answer]
Should be an easy one to solve here.

An asterisk * means 'ANY' so where you think you have 'backup every day as 4:00am' as:

`* 4 * * * `

This means 'backup every day when the hour starts with 4' I.e, every minute.4:00, 4:01, 4:02 etc

What you'll need is to set:

`0 4 * * *`

This will then backup only when the minute is 0 and the hour is 4.

The same goes for your 5:00am backup task. 

Use[ crontab.guru](https://crontab.guru/#*_4_*_*_*) to help with setting triggers for cron jobs :)

**Reply by DominArsen** [reply]
Hello, Thank you, it works flawlessly!

Bonjour, Merci cela fonctionne impecable !

**Reply by EugeneGGHQ** [reply]
De rien! 🙂
